internal/runners/clean/run_win.go

```diff
@@ -129,3 +132,44 @@ func removePaths(logFile string, paths ...string) error {
 
 	return nil
 }
+
+func checkAdmin() error {
+	doAdminCheck := true
+	installedAsAdmin, err := getAdminInstall()
+	if err != nil {
+		doAdminCheck = false
+		multilog.Error("Could not check if initial installation was run as admin, error: %v", err)
```

Comment:
This seems like an actual error, let's return it.

Reply:
I guess auto-update will create the registry key even for old installations of the state tool which is the use case I was trying to be mindful of.

Reply:
Though this will mean that if the key isn't there for whatever reason then the user will not be able to uninstall

Reply:
The error handling for auto updating is centralized in the auto updating mechanic though, you don't have to worry about that here.

The key not being there should probably be a use-case you handle explicitly. But `err != nil` is too vague in that regard.